lte/gateway/c/core/oai/include/s8_messages_types.h

```diff
@@ -53,6 +53,9 @@ typedef struct s8_create_session_response_s {
                                               // removed and shall update
                                               // imsi_ue_context_htbl based on
                                               // received sgw_s8_teid
+  teid_t sgw_s8_up_teid;  // Orc8r generates sgw's s8 interface's user plane
```

Comment:
nit `Orc8r generates s8 user plane TEID for SGW and shares it in create session response for default bearer`